repo/repo.go

```diff
@@ -357,6 +357,30 @@ func (r *Repo) GetRecord(ctx context.Context, rpath string) (cid.Cid, cbg.CBORMa
 	return cc, rec, nil
 }
 
+func (r *Repo) GetRecordBytes(ctx context.Context, rpath string) (cid.Cid, *[]byte, error) {
```

Comment:
`GetRecord()` could call this method to de-dupe some code, though I wouldn't hold everything up on it.